services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/configuration/Configuration.kt

```diff
@@ -168,26 +166,6 @@ class Configuration {
             tormaystarkasteluDao
         )
 
-    @Bean
-    fun hankeAttachmentsService(
```

Comment:
These are removed because they have Service annotation and no need to create the beans like this.

Reply:
If the `@Bean`s in `IntegrationTestConfiguration` are used preferably over the `@Service` bean, we could drop most of the stuff in this file.